conda_smithy/configure_feedstock.py

```diff
@@ -45,13 +45,18 @@ def render_run_docker_build(jinja_env, forge_config, forge_dir):
                     cases_not_skipped.append(vars + sorted(pkgs))
         matrix = sorted(cases_not_skipped, key=sort_without_target_arch)
 
-    target_fname = os.path.join(forge_dir, 'ci_support', 'run_docker_build.sh')
     if not matrix:
         # There are no cases to build (not even a case without any special
         # dependencies), so remove the run_docker_build.sh if it exists.
         forge_config["circle"]["enabled"] = False
-        if os.path.exists(target_fname):
-            os.remove(target_fname)
+
+        target_fnames = [
+            os.path.join(forge_dir, 'ci_support', 'run_docker_build.sh'),
+            os.path.join(forge_dir, 'ci_support', 'checkout_merge_commit.sh'),
+        ]
+        for each_target_fname in target_fnames:
+            if os.path.exists(each_target_fname):
+                os.remove(each_target_fname)
```

Comment:
Modified this code to handle removal of `ci_support/run_docker_build.sh` and  `ci_support/checkout_merge_commit.sh` when Linux builds are disabled. Tested this locally to make sure it worked.